mat64/dense.go

```diff
@@ -5,7 +5,10 @@
 package mat64
 
 import (
+	"bytes"
+
 	"github.com/gonum/blas/blas64"
+	"github.com/gonuts/binary"
```

Comment:
@btracey : gonuts/binary is mine. I can easily vendor it under, say, gonum/internal/binary